test/unit/specs/fields/fieldCleave.spec.js

```diff
@@ -41,9 +42,9 @@ describe("fieldCleave.vue", function() {
 			expect(input).to.be.defined;
 			expect(input.type).to.be.equal("text");
 			expect(input.classList.contains("form-control")).to.be.true;
-			expect(input.placeholder).to.be.equal(schema.placeholder);	
-			expect(input.readOnly).to.be.false;	
-			expect(input.disabled).to.be.false;	
+			// expect(input.placeholder).to.be.equal(schema.placeholder);
+			// expect(input.readOnly).to.be.false;
```

Comment:
Yeah, I wasn't sure you wanted to delete them or not, so meanwhile, I commented them. I will clean things now !